src/Nirum/Targets/Python.hs

```diff
@@ -299,6 +300,31 @@ toIndentedCodes :: (a -> T.Text) -> [a] -> T.Text -> T.Text
 toIndentedCodes f traversable concatenator =
     T.intercalate concatenator $ map f traversable
 
+compileParameters :: (ParameterName -> ParameterType -> Code)
+               -> [(T.Text, Code)]
+               -> Code
+compileParameters gen nameTypePairs =
+    toIndentedCodes (uncurry gen) nameTypePairs ", "
+
+compileFieldInitializers :: DS.DeclarationSet Field -> CodeGen Code
+compileFieldInitializers fields = do
+    initializers <- forM (toList fields) compileFieldInitializer
+    return $ T.intercalate "\n     " initializers
+  where
+    compileFieldInitializer :: Field -> CodeGen Code
+    compileFieldInitializer (Field fieldName' fieldType' _) =
+        case fieldType' of
+            SetModifier _ -> return [qq|frozenset($attributeName)|]
+            ListModifier _ -> do
+                let imports = [("nirum.datastructures", ["List"])]
```

Comment:
It prevents Nirum types to be named as `list` since it can make name conflict.  Seems `nirum.datastructures` need to export aliases e.g. `list_type`, `map_type`, etc.